### PR TITLE
Zowe Explorer release notes and doc updates

### PR DESCRIPTION
**Pull request by IgorCATech**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [x] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

Adding the Zowe Explroer 1.14 relaese notes to Zowe 1.21. Updating the Zowe Explorer usage section - new feature "Issue TSO Command".

:heart:Thank you!

### CODE REVIEW

File: docs/user-guide/ze-usage.md [hunk 1]
```diff
@@ -353,15 +353,26 @@ You can use the following functionalities when interacting with jobs:
...
 #### Issue MVS commands
 
-Zowe Explorer also enables you to issue MVS commands. You can issue such commands as Allocate or Exec against a profile.
+1. Right-click on your profile and select the **Issue MVS Command** option. Select the **Zowe:Issue MVS Command** option.
 
-1. Press the **F1** key on your keyboard.
-2. Select the **Zowe:Issue MVS Command** option.
-3. Select your profile.
-4. Issue an MVS command.
+   Alternatively, press the **F1** key on your keyboard, then select "the **Zowe:Issue MVS Command** option.
```
Comment (jellypuno) [inline docs/user-guide/ze-usage.md:362]: I would change "then select" to "search for **Zowe: Issue MVS Command**" 
If you haven't used this command before, you won't be able to see it immediately.

File: docs/user-guide/ze-usage.md [hunk 2]
```diff
@@ -353,15 +353,26 @@ You can use the following functionalities when interacting with jobs:
...
+2. Select your profile.
+3. Issue an MVS command.
 
 <img src="../images/ze/ZE-Jobs-Issue-TSO-Command.gif" alt="Issue a MVS command">
+
+#### Issue TSO commands
+
+1. Right-click on your profile and select the **Issue TSO Command** option. Select the **Zowe:Issue TSO Command** option.
+
+   Alternatively, press the **F1** key on your keyboard, then select "the **Zowe:Issue TSO Command** option.
```
Comment (jellypuno) [inline docs/user-guide/ze-usage.md:373]: same here

File: docs/user-guide/ze-usage.md [hunk 3]
```diff
@@ -353,15 +353,26 @@ You can use the following functionalities when interacting with jobs:
...
-Zowe Explorer also enables you to issue MVS commands. You can issue such commands as Allocate or Exec against a profile.
+1. Right-click on your profile and select the **Issue MVS Command** option. Select the **Zowe:Issue MVS Command** option.
 
-1. Press the **F1** key on your keyboard.
-2. Select the **Zowe:Issue MVS Command** option.
-3. Select your profile.
-4. Issue an MVS command.
+   Alternatively, press the **F1** key on your keyboard, then select "the **Zowe:Issue MVS Command** option.
+
+2. Select your profile.
```
Comment (jellypuno) [inline docs/user-guide/ze-usage.md:364]: No need to select a profile. It's either select a saved command or enter a new command

File: docs/user-guide/ze-usage.md [hunk 4]
```diff
@@ -353,15 +353,26 @@ You can use the following functionalities when interacting with jobs:
...
+3. Issue an MVS command.
```
Comment (jellypuno) [inline docs/user-guide/ze-usage.md:365]: Press enter to issue the command. Then it will be display in the output

**Review by jellypuno — CHANGES_REQUESTED** [review]

File: docs/user-guide/ze-usage.md (hunk 1 above)
Comment (IgorCATech) [reply docs/user-guide/ze-usage.md:362]: uh huh! Good catch

**Review by nannanli — APPROVED** [review]

**Pull request closed without merging**